api/jobs/params/update-job-parameters.ts

```diff
@@ -0,0 +1,69 @@
+import { BaseParameters } from "../../parameters/index";
+import { SmartlingException } from "../../exception/index";
+
+interface CustomFieldBasicRequest {
+    fieldUid: string,
+    fieldValue: string
+}
+
+export { CustomFieldBasicRequest };
+
+export class UpdateJobParameters extends BaseParameters {
```

Comment:
Maybe extend `CreateJobParameters` from `UpdateJobParameters` to avoid copy-pasting of same methods?